### ISSUE TITLE
MONDO:0008431 & MONDO:0018382 [Merge]

### ISSUE THREAD

**Issue by kanems** [monarch-initiative/mondo#7143]
**Mondo term (ID and Label)**
MONDO:0008431 slipped femoral capital epiphyses

**Reason for deprecation**
UMLS & MeSH treat these as synonyms D060048 MSH & C0149887
Some of the synonyms seem to be overlapping (or nearly so: "slipped capital femoral epiphysis" from Orphanet) 

**Term to be merged with**
MONDO:0018382 epiphysiolysis of the hip
**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

**Comment by nicolevasilevsky** [comment]
note to self:
`sh run.sh owltools --use-catalog mondo-edit.obo  --obsolete-replace MONDO:0008431 MONDO:0018382 -o -f obo mondo-edit.obo`

**Comment by kanems** [comment]
@nicolevasilevsky I am not sure what I need to review? I googled the phrase "epiphysiolysis of the hip" and saw multiple pages matching "Slipped Capital Femoral Epiphysis" like https://www.hopkinsmedicine.org/health/conditions-and-diseases/slipped-capital-femoral-epiphysis . And also noting that Orphanet uses these terms as syns. https://www.orpha.net/en/disease/detail/399329 
So I still think these can/should be merged. I am agnostic regarding the direction of the merge and Mondo's preferred name.

**Comment by nicolevasilevsky** [comment]
Oops, sorry @kanems! I meant to tag @katiermullen!
Thanks, as always, for reporting these issues!

**Comment by katiermullen** [comment]
I agree with the merge of MONDO:0008431 slipped femoral capital epiphyses into MONDO:0018382 epiphysiolysis of the hip based on the overlapping synonyms mentioned above.

**Comment by nicolevasilevsky** [comment]
@katiermullen do you want to try to do this merger? Or shall I?

**Comment by katiermullen** [comment]
I'll try it today and let you know if I have any problems. Thank you!
